Dockerfile

```diff
@@ -0,0 +1,36 @@
+# This file is part of REANA.
+# Copyright (C) 2017 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation; either version 2 of the License, or (at your option) any later
+# version.
+#
+# REANA is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# REANA; if not, write to the Free Software Foundation, Inc., 59 Temple Place,
+# Suite 330, Boston, MA 02111-1307, USA.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+
+FROM python:2.7-alpine
+
+ADD . /code
+WORKDIR /code
+
+RUN pip install -e .
+
+RUN adduser -u 1000 -D -g '' reanauser && \
+    chown -R reanauser:reanauser /code
+USER reanauser
+
+ENV REANA_DEMO_NAME="World"
```

Comment:
Based on the discussion in `reana` chat I think we should stick to an example without using env variables, something more in the line of [yadage hello world example](http://yadage.readthedocs.io/en/latest/#hello-world-example) and leave env vars for the future.
Either we provide the entire command to the image as `python -m reana_demo_helloworld -p name -o output/output.txt` or we specify `python -m reana_demo_helloworld` as a `docker` `entrypoint` and then just provide `-p` and `-o` along with its values.